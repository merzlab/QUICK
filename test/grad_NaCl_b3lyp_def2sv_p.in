B3LYP  BASIS=def2-sv(p) cutoff=1.0e-9 denserms=1.0e-6  zmake GRADIENT DIPOLE CHARGE=0

Na        -6.71296        1.52486        0.00000
Cl        -4.31558        1.52486        0.00000


