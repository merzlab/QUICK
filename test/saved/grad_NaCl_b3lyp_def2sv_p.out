  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2022                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **             This software is subject to the terms of the Mozilla Public License, v. 2.0.            **
  **                                                                                                     **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Shajan, A.; Giese, T. J.; Cruzeiro, V.W.D.; Smith, J.; Miao, Y.; He, X.; Ayers, K.;      
  Brothers, E.; Goetz, A.W.; Merz, K.M. QUICK-22.03                                                        
  University of California San Diego, CA and Michigan State University, East Lansing, MI, 2022             
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Fri Mar 17 12:16:55 2023
| INPUT FILE :    grad_NaCl_b3lyp_def2sv_p.in
| OUTPUT FILE:    grad_NaCl_b3lyp_def2sv_p.out
| DATA FILE  :    grad_NaCl_b3lyp_def2sv_p.dat
| BASIS SET PATH: /mnt/home/shajanak/QUICK_OG/basis

 @ Read Job And Atom

  KEYWORD=B3LYP  BASIS=DEF2-SV(P) CUTOFF=1.0E-9 DENSERMS=1.0E-6  ZMAKE GRADIENT DIPOLE CHARGE=0                                                                                                                   
 ============== JOB CARD =============
 METHOD = DENSITY FUNCTIONAL THEORY
 DENSITY FUNCTIONAL = B3LYP
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 READ SAD GUESS FROM FILE
 Z-MATRIX CONSTRUCTION
 DIPOLE
 STANDARD GRID = SG1
 GRADIENT CALCULATION
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE =    3
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-09
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-05
      GRADIENT CUTOFF  =  0.100E-06
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = DEF2-SV(P),  TYPE = CARTESIAN
| BASIS FILE = /mnt/home/shajanak/QUICK_OG/basis/DEF2-SV_P.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = NA
 BASIS FUNCTIONS =   16
 For Atom Kind =    2
 ELEMENT = CL
 BASIS FUNCTIONS =   19

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =    2    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    0    NUMBER OF NON-HYDROGEN ATOM =    2
 NUMBER OF ELECTRONS     =   28

 -- INPUT GEOMETRY -- :
    NA         -6.7130       1.5249       0.0000
    CL         -4.3156       1.5249       0.0000

 -- DISTANCE MATRIX -- :
              1           2
      1     0.00000
      2     4.53039     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   35
 NSHELL =   15 NPRIM  =   71
 JSHELL =   15 JBASIS =   35


 @ End Reading Molecular Information 


 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-06
   INITIAL GRID POINTS  =        7520
|   FINAL GRID POINTS    =        5108
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        2596
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =        2957

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1   -622.286557158      ------         0.243     1        0.00      0.24      0.00    0.6206E+00  0.3024E-01  0.3425E+00
|  2   -622.345371016  0.588139E-01       0.254     2        0.00      0.25      0.00    0.1168E+00  0.4017E-01  0.5547E+00
|  3   -622.345160291  -.210725E-03       0.251     3        0.00      0.25      0.00    0.1120E+00  0.1968E-01  0.3272E+00
|  4   -622.389812819  0.446525E-01       0.248     4        0.00      0.25      0.00    0.5063E-02  0.1404E-02  0.1984E-01
|  5   -622.389885121  0.723025E-04       0.240     5        0.00      0.24      0.00    0.2977E-02  0.3603E-03  0.4823E-02
|  6   -622.389902549  0.174271E-04       0.233     6        0.00      0.23      0.00    0.5127E-03  0.1069E-03  0.1676E-02
|  7   -622.389903351  0.802188E-06       0.224     7        0.00      0.22      0.00    0.1354E-04  0.3215E-05  0.3012E-04
|  8   -622.389903352  0.145749E-08       0.201     8        0.00      0.20      0.00    0.1071E-05  0.6388E-06  0.6804E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   8 CYCLES
| MAX ERROR = 0.107135E-05   RMS CHANGE = 0.638782E-06   MAX CHANGE = 0.680363E-05
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =    13.9999379893
 BETA ELECTRON DENSITY     =    13.9999379893
 ELECTRONIC ENERGY    =   -663.666688011
 CORE_CORE REPULSION  =     41.276784659
 TOTAL ENERGY         =   -622.389903352

 @ End Energy calculation


 @ Begin Gradient Calculation

 ANALYTICAL GRADIENT: 
---------------------------------------------------------
   ATOM         X                Y                Z
---------------------------------------------------------
    1    -0.0078405461     0.0000000024    -0.0000000038
    2     0.0078405461    -0.0000000024     0.0000000038
---------------------------------------------------------

 @ End Gradient Calculation


 @ Begin Zmake conversion

  
 Z-MATRIX:

   ATOM  ELEMENTAL  BOND LENGTH    BOND ANGLE    DIHEDRAL ANGLE
  NUMBER  SYMBOL    (ANGSTROMS)    (DEGREES)       (DEGREES)
     I                 I-NA         I-NA-NB        I-NA-NB-NC      NA   NB   NC

     1      NA       0.00000  0     0.00000  0      0.00000  0      0    0    0
     2      CL       2.39738  1     0.00000  0      0.00000  0      1    0    0

 @ End Zmake conversion


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   NA             0.5087             0.2400
   CL            -0.5087            -0.2400
   TOTAL         0.0000             0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
   -8.1093   -0.0000   -0.0000    8.1093

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIALIZATION TIME =     0.002357000(  0.07%)
| INITIAL GUESS TIME  =     0.255202000(  7.47%)
| OVERLAP 1e INTEGRALS TIME      =     0.000838000( 0.02%)
| OVERLAP 1e DIAGONALIZATION TIME =     0.000374000( 0.01%)
| DFT GRID OPERATIONS =     0.038252000(  1.12%)
|       TOTAL GRID FORMATION TIME   =     0.000365000(  0.01%)
|       TOTAL GRID WEIGHT TIME      =     0.000987000(  0.03%)
|       TOTAL OCTREE RUN TIME       =     0.020000000(  0.59%)
|       TOTAL PRESCREENING TIME     =     0.010000000(  0.29%)
|       TOTAL DATA PACKING TIME     =     0.006900000(  0.20%)
| TOTAL SCF TIME      =     1.896990000( 55.51%)
|       TOTAL OP TIME      =     1.881377000( 55.06%)
|             TOTAL 1e TIME      =     0.002390000(  0.07%)
|                KINETIC 1e INTEGRALS TIME    =     0.001994000(  0.06%)
|                ATTRACTION 1e INTEGRALS TIME =     0.000388000(  0.01%)
|             TOTAL 2e TIME      =     1.391324000( 40.72%)
|             TOTAL EXC TIME     =     0.485114000( 14.20%)
|             TOTAL ENERGY TIME  =     0.000063000(  0.00%)
|       TOTAL DII TIME      =     0.015469000(  0.45%)
|             TOTAL DIAG TIME    =     0.002929000(  0.09%)
| TOTAL GRADIENT TIME      =     1.220993000( 35.73%)
|       TOTAL NUCLEAR GRADIENT TIME =     0.000003000(  0.00%)
|       TOTAL 1e GRADIENT TIME      =     0.010954000(  0.39%)
|          KINETIC 1e GRADIENT TIME      =     0.010954000(  0.32%)
|          ATTRACTION 1e GRADIENT TIME   =     0.002249000(  0.07%)
|       TOTAL 2e GRADIENT TIME      =     1.065051000(31.17%)
|       TOTAL EXC GRADIENT TIME     =     0.142736000(  4.18%)
| DIPOLE TIME        =     0.002508000( 0.07%)
| TOTAL TIME          =     3.417169000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes  3.4 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Fri Mar 17 12:16:58 2023
